environment.yml

```diff
@@ -1,9 +1,11 @@
 name: quicknxs
 channels:
+    - neutrons/label/main
     - mantid/label/main
+    - oncat
     - conda-forge
 dependencies:
-- mantidworkbench=6.12.0
+- mr_reduction=2.2.0
```

Comment:
There should be no difference in mr_reduction dependency between the conda recipe and the conda environment (one is `=2.20` and the other is `>=2.2`).

At some point in the future, we're going to forget about setting the dependency in the conda recipe. I strongly suggest you adopt a similar solution to the one implemented in RefRed to make sure the [dependencies of the conda recipe and environment file are synced](https://github.com/neutrons/RefRed/blob/next/.github/workflows/test-and-deploy.yml#L29C31-L29C51).

Reply:
Good idea, thanks for the link :+1: I added a similar check.